tslint.json

```diff
@@ -20,7 +20,7 @@
     "no-bitwise": true,
     "no-shadowed-variable": true,
     "no-unused-expression": true,
-    "no-unused-variable": true,
+    "no-unused-variable": [true, {"ignore-pattern": "ddescribe|iit|.*_$"}],
```

Comment:
This was actually my suggestion. From a dev experience perspective, you frequently find yourself wanting to swap stuff between `describe` and `ddescribe` / `it` and `iit` (I even have a [WebStorm plugin](https://plugins.jetbrains.com/plugin/7233?pr=idea) that does it). If you have to update the imports every time to swap these, it gets annoying. 

This doesn't have any cost for the library itself since these are only in unit tests.